src/idpi/cli.py

```diff
@@ -21,7 +29,113 @@ def print_version(ctx, _, value: bool) -> None:
     expose_value=False,
     callback=print_version,
 )
-@click.group()
 def main() -> None:
     """Console script for test_cli_project."""
     print("CLI for IDPI")
+
+
+RESAMPLING = {
+    "nearest": regrid.Resampling.nearest,
+    "bilinear": regrid.Resampling.bilinear,
+    "cubic": regrid.Resampling.cubic,
+}
+
+
+def _load_mapping():
+    mapping_path = files("idpi.data").joinpath("field_mappings.yml")
+    return yaml.safe_load(mapping_path.open())
+
+
+def handle_vector_fields(ds):
```

Comment:
This function should check that the bit 5 of the resolution and components flag is set on the input field

Reply:
Or is it not implemented yet? In which case I would add a comment in the code

Reply:
It is implemented in the last commit: https://github.com/MeteoSwiss-APN/icon_data_processing_incubator/pull/115/commits/9836bb2c05e96c06c1c12313c00ca42cb70bd6dd